Tests/GRPCTests/ConnectionPool/GRPCChannelPoolTests.swift

```diff
@@ -618,5 +623,24 @@ final class GRPCChannelPoolTests: GRPCTestCase {
 
     XCTAssertGreaterThan(statsEvents.count, 0)
   }
+
+  #if canImport(Network)
+  func testNWParametersConfigurator() {
+    let counter = NIOLockedValueBox(0)
+    self.group = NIOTSEventLoopGroup()
```

Comment:
We're somewhat skirting around how these test helpers were designed which could lead to some surprises. 

I think the more appropriate solution is to add a flag to `configureEventLoopGroup` which allows you to toggle whether an MTELG is used vs. a NIOTSELG, that way the other helpers should work as expected (e.g. `setUpClientAndServer`)